examples/dom_example.re

```diff
@@ -20,79 +11,79 @@ let unwrapUnsafely =
 /*
  * These SHOULD type check
  */
-document |> Document.createElement("div")
-         |> Element.className;
+let _ = document->Document.createElement("div")->Element.className;
 
-document |> Document.createElement("div")
-         |> Element.nextElementSibling
-         |> map(Element.innerText); /* innerText is a function that accepts a Node */
+let _ =
+  document->Document.createElement("div")->Element.nextElementSibling->map(Element.innerText); /* innerText is a function that accepts a Node */
 
 /* Before subtyping:
-document |> Document.createElement("div")
-         |> Element.asNode
-         |> Node.parentElement /* inherited from Node, returns DomRe.element */
-         |> map(Element.innerText); /* inherited from Node */
-*/
+   document -> Document.createElement("div")
+            -> Element.asNode
+            -> Node.parentElement /* inherited from Node, returns DomRe.element */
+            -> map(Element.innerText); /* inherited from Node */
+   */
 
 /* After subtyping: */
-document |> Document.createElement("div")
-         |> Element.parentElement /* inherited from Node, returns DomRe.element */
-         |> map(Element.innerText); /* inherited from Node */
+let _ =
+  document
+  ->Document.createElement("div")
+  ->Element.parentElement /* inherited from Node, returns DomRe.element */
+  ->map(Element.innerText); /* inherited from Node */
 
-let el =
-  document |> Document.createElement("div")
-           |> Element.asHtmlElement
-           |> unwrapUnsafely;
+let el = document->Document.createElement("div")->Element.asHtmlElement->unwrapUnsafely;
 
 /*
-document |> Document.asHtmlDocument
-         |> andThen(HtmlDocument.body)
-         |> map(Element.appendChild(el));
-*/
+ document -> Document.asHtmlDocument
```

Comment:
Maybe reformat these to look more what refmt would produce without the spaces around the `->` operator.

Reply:
ah yeah I forgot about the commented code